docs/translations/spanish/apps/fundraising.md

```diff
@@ -0,0 +1,101 @@
+
+# [Fundraising](https://github.com/aragon/aragon-apps/tree/master/apps/fundraising)
+
+_**Code in Github:**_ [aragon-apps/apps/fundraising](https://github.com/aragon/aragon-apps/tree/master/apps/fundraising)
+
+The Fundraising app permite vender tokens de la organización a cambio de otros tokens. Las ventas que se pueden realizar son ventas limitadas simples con precios variables a lo largo del tiempo. Al ajustar algunos parámetros, se pueden lograr diferentes dinámicas de venta.
+
+Los tipos de venta más complejos se pueden implementar como aplicaciones de terceros.
+
+### Dependencias y permisos
+
+- `Token Manager`: En la inicialización requiere una referencia a un `Token Manager` ejemplo. Uno fundraising app ejemplo solo puede 'vender' el token subyacente administrado por su `Token Manager`. **The fundraising app 
+se deben otorgar permisos para tokens de menta en el Token Manager**. The Token Manager debe haber sido inicializado en `Native mode` para que pueda mentir arbitrariamente nuevos tokens (See [Token Manager spec](./token-manager)).
+
+- `Vault`: Una referencia a un `vault` también se requiere en la inicialización. Sin embargo, no hay interacción directa con el vault, solo se usa como la dirección donde se depositan los ingresos de la venta (vanilla ERC20 transfers).
+
+Ambos parámetros no son modificables después de la inicialización.
+
+### Definiciones
+
+- **Inverse pricing**: Dado que no existe soporte nativo para los números de coma flotante en Solidity, tenemos que introducir el concepto de fijación de precios inversa para permitir precios inferiores a 1. Los débitos simbólicos deben tenerse en cuenta al establecer los precios.
+	- cuando inverse pricing es `false` en una venta, el precio significa cuántos tokens de entrada se deben pagar por cada token que se compra. Eso significa que la cantidad de tokens comprados se calcula como`boughtTokens = payedTokens / price`.
+	- cuando inverse pricing es `true`, precio significa cuántos tokens comprados se adquieren por token de entrada. `boughtTokens = payedTokens * price`
+
+- **Price periods**:
+	- Una venta pasará por uno o múltiples períodos durante su ciclo de vida.
+	- Los períodos determinan cuál es el precio en un momento dado durante la venta.
+	- Un período comienza en la misma marca de tiempo cuando finalizó el período anterior (o en el caso del primero, cuando comienza la venta).
+	- Un período tiene un precio inicial y final. Si no son lo mismo, el precio de una marca de tiempo determinada se interpola linealmente en función del tiempo.
+	- Las ventas pueden tener hasta 50 períodos.
+
+![](./rsc/fundraising_periods.png)
```

Comment:
`![](../../../dev/apps/rsc/fundraising_periods.png)`

Reply:
@Smokyish correct. We can use rsc/fundraising_periods.png as it's not translated.

Regarding the links in the original version also, i am not able to get the pages properly. That's why i kept same as we have to fix at both places ( https://github.com/aragon/aragon-wiki/blob/master/docs/dev/apps/index.md  )

Sorry i didn't got regarding docs/translations/spanish/apps/fundraising.md

Please clarify me once. 

Thanks

Reply:
@linuxchip The link did work if you used it in https://wiki.aragon.one/dev/apps/ but it was erroneous, i've fixed those links now to be `.md` as they should and they are working in GH as well.

Reply:
@Smokyish thanks i will fix in spanish

What about my other question ? Anything to be fixed here ?

" Sorry i didn't got regarding docs/translations/spanish/apps/fundraising.md "

Thanks in advance

Reply:
@Smokyish  i fixed links and deleted using seperate fundraising_periods.png file